config/site_settings.yml

```diff
@@ -1778,6 +1778,12 @@ trust:
     default: 0
     client: true
     enum: "TrustLevelSetting"
+  profile_background_allowed_groups:
+    default: "3|10" # auto group staff and trust_level_0
+    type: group_list
+    allow_any: false
+    refresh: true
+    validator: "AtLeastOneGroupValidator"
```

Comment:
Do we need at least one group? `in_trust_level_to_allow_user_card_background:` is allowing no groups to disable feature completely

Reply:
Yeah you're right, will add :+1: